Dockerfile

```diff
@@ -76,7 +76,7 @@ ENTRYPOINT ["/root/start.sh"]
 
 
 # Build beegfs-all docker image with `docker build -t repo/image-name  --target beegfs-all .`  
-FROM base AS beegfs-all
+FROM --platform=$TARGETPLATFORM base AS beegfs-all
 ARG BEEGFS_SERVICE="beegfs-all"
 RUN apt-get update && apt-get install libbeegfs-ib beegfs-mgmtd beegfs-meta beegfs-storage -y && rm -rf /var/lib/apt/lists/*
```

Comment:
When the packages are downloaded here does it get downloaded for `amd64` or `arm64v8`.
Usually, the base architecture where the docker image was built was checked and only those packages were installed.

Reply:
I see both packages getting downloaded in logs